submissions/emmanuelEsuong/htmlForm/signupPage.html

```diff
@@ -1,88 +1,130 @@
 <!DOCTYPE html>
-
 <html lang="en">
-
+    <!--header-->
 <head>
     <meta charset="UTF-8">
     <title>SignUp</title>
+        <!--links-->
+        <link rel="stylesheet" type="text/css" href="../css/styles.css">
+        <link rel="stylesheet" type="text/css" media="screen" href="../css/styles-mediaQuery.css">
+        <link rel="stylesheet" href="https://cdn.rawgit.com/vaakash/socializer/80391a50/css/socializer.min.css">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
You can't use bootstrap or any other form css to style, just your own. Remove this and everywhere else you have applied their styles. @ElJentle

Reply:
Please, how do I add social media icon links to the pages?